src/components/CheckboxSet.js

```diff
@@ -44,27 +45,29 @@ class CheckboxSet extends Component {
       <Consumer>
         {context => (
           <List>
-            {constants[this.props.filterName].map((option, index) => (
-              <ListItem key={index}>
-                <Checkbox
-                  checked={
-                    context.state.filters[this.props.filterName].indexOf(
-                      option
-                    ) != -1
-                  }
-                  label={option}
-                  value={option}
-                  id={`${this.makeId(option)}`}
-                  name={`${this.makeId(option)}`}
-                  handleChange={e =>
-                    context.actions.getCheckboxSetValues(
-                      e,
-                      this.props.filterName
-                    )
-                  }
-                />
-              </ListItem>
-            ))}
+            {constants[this.props.filterName].map(option => {
+              return (
+                <ListItem key={shortid.generate()}>
```

Comment:
Nit: these will be remounted on every render. It won't cause perf issues in this case, but if we weren't pressed for time I'd recommend using `this.makeId(option)` as the key here too.